sources/host-ctr/cmd/host-ctr/all_devices_allowed.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/containers"
+	"github.com/containerd/containerd/oci"
+	runtimespec "github.com/opencontainers/runtime-spec/specs-go"
```

Comment:
nit: if we import this as "specs" then it will get rid of most of the delta from the source